<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="procframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>680</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="stateframe">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>231</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>211</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="hostnamelabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="activateTimelabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="RunTimelabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="CurrentTimelabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MEM</string>
     </property>
    </widget>
    <widget class="QLabel" name="cpuphereLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="memphereLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>350</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Oslabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>341</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuproc">
    <property name="title">
     <string>proc</string>
    </property>
    <addaction name="actionshow_ps"/>
   </widget>
   <widget class="QMenu" name="menucpu">
    <property name="title">
     <string>cpu</string>
    </property>
    <addaction name="actioninfo"/>
   </widget>
   <widget class="QMenu" name="menuswap">
    <property name="title">
     <string>Graph</string>
    </property>
    <addaction name="action_Graph"/>
   </widget>
   <widget class="QMenu" name="menuother">
    <property name="title">
     <string>other</string>
    </property>
    <addaction name="NewProgressaction"/>
    <addaction name="PowerOffaction"/>
   </widget>
   <addaction name="menuproc"/>
   <addaction name="menucpu"/>
   <addaction name="menuswap"/>
   <addaction name="menuother"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actioninfo">
   <property name="text">
    <string>info</string>
   </property>
  </action>
  <action name="actioncpu">
   <property name="text">
    <string>cpu使用率的图形化显示</string>
   </property>
  </action>
  <action name="action_Graph">
   <property name="text">
    <string>使用率的图形化显示</string>
   </property>
  </action>
  <action name="NewProgressaction">
   <property name="text">
    <string>用新进程运行一个其他程序</string>
   </property>
  </action>
  <action name="PowerOffaction">
   <property name="text">
    <string>关机功能</string>
   </property>
  </action>
  <action name="actionshow_ps">
   <property name="text">
    <string>show ps</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
